packages/cli/test/upload-url-hash.test.js

```diff
@@ -1,5 +1,5 @@
 /**
- * @license Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
please revert - as a policy we don't update these years

Reply:
@connorjclark - Done and I have squashed the commits as well